runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -125,6 +133,9 @@
 	private static final String SUPER_ANTIVENOM_DRINK_MESSAGE = "You drink some of your super antivenom potion";
 
 	private int lastRaidVarb;
+	private int lastWorldX;
+	private int lastWorldY;
+	private HashSet<GameTimer> freezeTimers;
```

Comment:
Change HashSet<> to Set<> and use ImmutableSet.of here, and change this to static final FREEZE_TIMERS